voltixApp/VoltixApp/Views/Components/VaultItem.swift

```diff
@@ -92,8 +92,8 @@ struct VaultItem: View {
 						{
 							presentationStack.append(.bitcoinTransactionsListView(SendTransaction(coin: coin)))
                         } else if coin.chain.name.lowercased() == "ethereum" {
-                            if !coin.contractAddress.isEmpty {
-                                presentationStack.append(.erc20TransactionsListView(coin.contractAddress))
+							if coin.isNativeToken {
```

Comment:
```suggestion
							if !coin.isNativeToken {
```

Reply:
Do you mean that Native token is ETH, BTC, and not ERC20 tokens?

Reply:
On ETH blockchain , only ETH is the native token , it is the gas token , all other ERC20 tokens are not